index.html

```diff
@@ -2559,7 +2587,7 @@ <h4>
           <ol>
             <li>For each <var>connection</var> in the <a>set of controlled
             presentations</a> that was connected to <var>P</var>, <a>queue a
-            task</a> to run the following steps:
+            Presentation API task</a> to run the following steps:
```

Comment:
Same update as that done in "Terminating a presentation in a controlling browsing context" algorithm:

```suggestion
            global task</a> on the <a>presentation task source</a> given
            <var>connection</var>'s <a>relevant global object</a> to run the
            following steps:
```

Reply:
Done